react/components/molecules/toggle-button/toggle-button.js

```diff
@@ -0,0 +1,147 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin } from "../../../util";
+
+import { Button } from "../../atoms";
+
+export class ToggleButton extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            text: PropTypes.string,
+            icon: PropTypes.string,
+            iconSecondary: PropTypes.string,
+            color: PropTypes.string,
+            colorSecondary: PropTypes.string,
+            active: PropTypes.bool,
+            orientation: PropTypes.string,
+            loading: PropTypes.bool,
+            disabled: PropTypes.bool,
+            showText: PropTypes.bool,
+            buttonIconProps: PropTypes.object,
+            onUpdateActive: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            text: undefined,
+            icon: undefined,
+            iconSecondary: undefined,
+            color: "#f4f5f7",
+            colorSecondary: "#4a6fe9",
+            active: false,
+            orientation: undefined,
+            loading: false,
+            disabled: false,
+            showText: false,
+            buttonIconProps: {},
+            onUpdateActive: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            activeData: props.active
+        };
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.active !== this.props.active) {
+            this.setState({
+                activeData: this.props.active
+            });
+        }
+    }
+
+    onPress = () => {

```

Comment:
`onPress` receives an event as argument.

Add that argument here, even if you aren't using it